charts/geoweb-presets-backend/README.md

```diff
@@ -132,12 +156,14 @@ The following table lists the configurable parameters of the Presets backend cha
 | `presets.db.POSTGRES_DB` | Default postgres database name | `presets` |
 | `presets.db.POSTGRES_USER` | Default postgres database user | `postgres` |
 | `presets.db.POSTGRES_PASSWORD` | Default postgres database password | `postgres` |
+| `presets.useCustomConfigurationFiles` | Use custom configurations | `false` |
+| `presets.customConfigurationLocation` | Where custom configurations are located *(local\|s3)* | `local` |
 | `presets.useCustomWorkspacePresets` | Use custom presets | `false` |
 | `presets.customWorkspacePresetLocation` | Where custom presets are located *(local\|s3)* | `local` |
-| `presets.volumeAccessMode` | Permissions of the application for the custom presets PersistentVolume used | `ReadOnlyMany` |
-| `presets.volumeSize` | Size of the custom presets PersistentVolume | `100Mi` |
 | `presets.customPresetsPath` | Path to the folder which contains custom presets | |
```

Comment:
Small detail: customConfigurationFolderPath is missing README entry here
Everything else seems very good!

Reply:
Added